common/views/components/MainViewerPrototype/MainViewerPrototype.tsx

```diff
@@ -335,11 +335,23 @@ const MainViewer: FunctionComponent<Props> = ({
     let currentCanvas;
     if (firstRenderRef.current) {
       setActiveIndex(canvasIndex);
-      mainViewerRef &&
-        mainViewerRef.current &&
-        mainViewerRef.current.scrollToItem(canvasIndex, 'start');
-      setFirstRender(false);
       currentCanvas = canvases && canvases[canvasIndex];
+      const viewer = mainViewerRef?.current;
+      const isLandscape = currentCanvas.width > currentCanvas.height;
+
+      if (isLandscape) {
```

Comment:
could we leave a note here, the one in the PR is super useful, but I imagine I won't be able to remember it when looking at this code in the future.